src/pybind/mgr/dashboard/controllers/cluster.py

```diff
@@ -19,3 +19,7 @@ def list(self):
                  parameters={'status': (str, 'Cluster Status')})
     def singleton_set(self, status: str):
         ClusterModel(status).to_db()
+
+    @RESTController.Collection('GET', 'capacity')
+    def get_df(self):
```

Comment:
Then why not calling this just `get_capacity` and carrying over 2 different names: capacity and df (which is a much obscure term).